topics/fair/tutorials/fair-ena/tutorial.md

```diff
@@ -0,0 +1,305 @@
+> 2. Similar to how you downloaded the sample checklist, download the Read submission template for 'paired reads using two fastq files'.
+> 3. Select the relevant optional fields to be included in the spreadsheet.
+> 
+> > <tip-title></tip-title>
+> > + Tick the box next to 'Show Descriptions' to reveal text describing the attributes
+> > + To find the sample alias and accession, look in 'Sample Reports'
+> > + To find the study accession, look in 'Studies Reports'  
+> {: .tip}
+> 
+> {:start="4"}
+> 4. Click 'Next' and download the spreadsheet .tsv file.
+> 5. Fill in the spreadsheet, paying particular attention to the following:
+> sample: same as you used in sample submission - alias or accession
+> study: use study accession
+> forward/reverse_file_name: ensure this is the same as what will be uploaded. It should be the compressed .gz file you created.
+> forward/reverse_file_md5: copy this from md5sums.tsv you generated earlier.
+> 6. Uploaded the completed template spreadsheet and submitted it.
+> 7. If metadata validation is successful a pop-up message bearing a run accession (ERR*) and experiment accession (ERX*) will appear for each read submission. 
+> 8. Confirm the processing status of your submission via Runs Report.
+> 9. Any errors generated in the process will be communicated to you on the registered email.
+>
+> <tip-title></tip-title>
+>  The run submission holds information about the raw read files generated in a run of sequencing as well as their location on an FTP server. The experiment submission holds metadata that describes the methods used to sequence the sample.
+> {: .tip}
+>
+{: .hands_on}
+
+## 5. Read data upload
+ENA offers [several methods](https://ena-docs.readthedocs.io/en/latest/submit/fileprep/upload.html) to achieve this. Please take a look at all the options should any suit your needs better than the one provided below.
+
+If you are using a Linux-based system you can use the below commands without needing to install any software.
+
+```
+cd /path/to/fastq.gz
+curl -T <<your_file>>.gz -u Webin-XXX:password ftp://webin2.ebi.ac.uk/
+```
+
+Check the processing status of your upload via the Run Processing Report.  
+
+> <warning-title></warning-title>
+> The Webin upload area is a temporary transit location which is not backed up. Always ensure you retain a local copy of the data till the files have been successfully submitted and archived. 
+{: .warning}
+
+## 6. Post-submission editing
+Metadata submitted for each object can be edited via the Reports (Studies, Samples, Runs) by clicking on the box-arrow icon in the Action column next to the table presenting the reports. The underlying xml of the submitted item can be inspected and edited. 
+
+However, for complex edits you might have to contact [ENA helpdesk](https://www.ebi.ac.uk/ena/browser/support). 
+
+This work was funded by the ELIXIR-UK: FAIR Data Stewardship training UKRI award (MR/V038966/1).

```

Comment:
So with the funding it should look like this, do you still want to include this line?

![image](https://github.com/galaxyproject/training-material/assets/458683/41a03ed7-7fd0-466b-ba39-8d3a62240b04)